bin/mkrc.in

```diff
@@ -122,3 +125,5 @@ for file in $files; do
   $VERBOSE "Linking..."
   $INSTALL -d "$DOTFILES_DIR" -t "${tag:--}" $install_args "$dotless"
```

Comment:
I worry about what changing `IFS` does to `$INSTALL` aka `rcup`. It iterates over space-separated things, such as `DOTFILES_DIRS`.

Would you mind adding a test for mkrc with multiple `DOTFILES_DIRS`?